### ISSUE TITLE
Add <count1> to properly check if a command count was set and isn't just the default

### ISSUE THREAD

**Issue by nhooyr** [neovim/neovim#13446]
See https://github.com/neovim/neovim/pull/13417

Looks like we don't have a `<count1>` for `<count>` like we have `v:count1` for `v:count`. The `count1` variable helps to check if a count was actually set or is just the default. The reasoning being when no count is provided and the default is used, `count1` will differ from `count` but when a count is provided, they will be the same.

Another option is to make `v:count` work for commands and not just keybinds. Though I'm concerned about backwards compatibility here.

**Comment by jamessan** [comment]
I think you're looking for `-count=1`.

**Comment by nhooyr** [comment]
The problem is 1 is a valid section for Man. So e.g. if you set `b:man_default_sects = '3,1'` and then did `Man printf`, it'd still open `printf` in section 1 instead of section 3 because you provided an explicit section.

**Comment by nhooyr** [comment]
It's definitely a minor quirk and may not be worth adding another variable for.

**Comment by jamessan** [comment]
You referenced `v:count1`, whose only difference from `v:count` is that it defaults to 1.  There's no way to know for sure whether the user supplied a value with either `v:count`/`v:count1` or `<count>`.  At least with the latter, you can specify the default value, so you could default to a value that isn't semantically valid and then know whether to use it.

**Comment by nhooyr** [comment]
> There's no way to know for sure whether the user supplied a value with either v:count/v:count1 or <count>. 

There is! The user supplied a value when they are both equal. If they are not equal, i.e count is 0 and count1 is 1, then the user did not supply a value.

**Comment by nhooyr** [comment]
Sorry, the above of course assumes the default count is 0.

**Comment by nhooyr** [comment]
Or any number not `1`.

**Comment by shade-of-noon** [comment]
I think a nicer alternative would be to introduce/allow an invalid count value, e.g.:
```
com! … -count=v:no_value …
```
which should evaluate to something different from any "valid" count… Or something to this effect, sorry I'm not familiar with ~~this part~~ most of the code.

**Comment by nhooyr** [comment]
Yes, that could be a better way to do it. Perhaps `-count=v:false`?
